### PR TITLE
Display map on mobile

### PR DESCRIPTION
**Pull request by faraznaeem**
## PT-Link https://www.pivotaltracker.com/story/show/150627329 ##

We added a map to the mobile app

### What we have learned: ###
- How to add a Google map to a mobile
- How to troubleshoot

### Screenshot ###
pasted image at 2017_08_31 01_46 pm

### CODE REVIEW

File: config.xml [hunk 1]
```diff
@@ -88,4 +83,5 @@
     <plugin name="cordova-plugin-statusbar" spec="2.2.2" />
     <plugin name="cordova-plugin-device" spec="1.1.4" />
     <plugin name="cordova-plugin-splashscreen" spec="~4.0.1" />
+    <plugin name="cordova-plugin-geolocation" spec="^2.4.3" />
```
Comment (sigu) [inline config.xml]: We don not need this plugin at this moment. To remove it `ionic cordova plugin rm cordova-plugin-geolocation`

File: package.json [hunk 2]
```diff
@@ -21,9 +21,11 @@
     "@angular/platform-browser": "4.1.3",
     "@angular/platform-browser-dynamic": "4.1.3",
     "@ionic-native/core": "3.12.1",
+    "@ionic-native/geolocation": "^4.2.1",
```
Comment (sigu) [inline package.json]: We dont need this too 😄   `npm uninstall --save @ionic-native/geolocation`

File: src/app/app.component.ts [hunk 3]
```diff
@@ -1,9 +1,10 @@
 import { Component, ViewChild } from '@angular/core';
-import { Nav, Platform } from 'ionic-angular';
+import { Nav, Platform, IonicPage } from 'ionic-angular';
```
Comment (sigu) [inline src/app/app.component.ts]: This import `IonicPage` is not required

File: src/pages/home/home.html [hunk 4]
```diff
@@ -1,18 +1,19 @@
+<!--
```
Comment (sigu) [inline src/pages/home/home.html]: Remove  the generated comments

**Review by sigu — CHANGES_REQUESTED** [review]

File: config.xml (hunk 1 above)
Comment (caressahh) [reply config.xml]: done!

File: package.json (hunk 2 above)
Comment (caressahh) [reply package.json]: GREAT! done!

File: src/app/app.component.ts (hunk 3 above)
Comment (caressahh) [reply src/app/app.component.ts]: fixed!

File: src/pages/home/home.html (hunk 4 above)
Comment (caressahh) [reply src/pages/home/home.html]: done!

**Review by diraulo — APPROVED** [review]

**Pull request merged**